dvc/lock.py

```diff
@@ -3,58 +3,61 @@
 from __future__ import unicode_literals
 
 import os
-import time
-import zc.lockfile
+from datetime import timedelta
+import hashlib
```

Comment:
Move this up, mising up `from` and `import` like this is not nice :)

Reply:
@Suor It should look like:

```
import os
import time
import haslib

from datetime import timedelta
```

Reply:
Why? We never separate that and there is no such practice outside dvc either.

Reply:
@Suor Just looks nicer, while we were at it. But ok, you are right, let's leave it as is.